### ISSUE TITLE
Feature Request: Force Voice Chat 

### ISSUE THREAD

**Issue by JaHollYT** [DiscordSRV/DiscordSRV#930]
Option in config called Force-Voice-Chat or something.
When this turned on: users have to be in lobby channel when they want to join.
If a player quits discord or leaves a voice channel he also get kicked from the server.

--> you can only play when player is connected to voice-syctem of discordsvr

**Comment by kotx** [comment]
What about players who haven't been linked? How will this be implemented?

**Comment by Scarsz** [comment]
> What about players who haven't been linked? How will this be implemented?

The framework for this already exists, this would be an addon to the `require a linked account to play` module so account linking wouldn't be an issue

**Comment by busterbeam** [comment]
I don't think this was properly answered.  Account linking is too simply to do preliminary setup. this user is asking that every time a user is in game he also has to be in the correct discord voice channel.. Forcing a player to use their voice... you could extend this idea to have some kind of voice captcha so they are encouraged to speak and not simply listen

**Comment by kotx** [comment]
I think a voice captcha would be too niche of a use case personally. 
Also, Discord's API doesn't officially support receiving voice, so this could be hard to implement. JDA also has it as low priority as well: https://github.com/DV8FromTheWorld/JDA/issues/904
